### PR TITLE
Update version of pymilvus and fix ci cases

### LINKED ISSUE #16310: [Bug]: Update pymilvus version
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version: latest
- Deployment mode(standalone or cluster):
- SDK version(e.g. pymilvus v2.0.0rc2):
- OS(Ubuntu or CentOS): 
- CPU/Memory: 
- GPU: 
- Others:
```

### Current Behavior

Update pymilvus version

### Expected Behavior

Update pymilvus version

### Steps To Reproduce

```markdown
Update pymilvus version
```

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by czs007**
Resolve: #16310 

Signed-off-by: zhenshan.cao <zhenshan.cao@zilliz.com>

### CODE REVIEW

File: tests/python_client/testcases/test_index.py [hunk 1]
```diff
@@ -442,8 +443,7 @@ def test_create_index_on_field(self, connect, collection, get_simple_index):
         """
         tmp_field_name = "int64"
         result = connect.insert(collection, default_entities)
-        with pytest.raises(Exception) as e:
-            connect.create_index(collection, tmp_field_name, get_simple_index)
+        connect.create_index(collection, tmp_field_name, get_simple_index)
```
Comment (yanliang567) [inline tests/python_client/testcases/test_index.py:446]: this case need a re-design, I think. @binbinlv

**Comment by yanliang567** [comment]
Related: #16310

File: tests/python_client/testcases/test_index.py (hunk 1 above)
Comment (binbinlv) [reply tests/python_client/testcases/test_index.py:446]: @yanliang567 OK, I will rewrite these scenario after this PR merged.

**Pull request merged**